client/packages/requisitions/src/ResponseRequisition/ListView/ListView.tsx

```diff
@@ -55,6 +55,12 @@ export const ResponseRequisitionListView: FC = () => {
             getRequisitionTranslator(t)(status as RequisitionNodeStatus),
         },
       ],
+      // Would ideally be a status (combination of requisition.status and requisition.authorisationStatus) ?
```

Comment:
I think this column would also be behind preference, ohh this would be the use case where the other preference that was deleted would be used, sorry if i asked for it to be removed earlier